implementations/micrometer-registry-elastic/src/test/java/io/micrometer/elastic/ElasticsearchMeterRegistryElasticsearch5IntegrationTest.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2019 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.elastic;
+
+import org.junit.jupiter.api.Tag;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+/**
+ * Integration tests on {@link ElasticMeterRegistry} for Elasticsearch 5.
+ *
+ * @author Johnny Lim
+ */
+@Testcontainers
+@Tag("docker")
+class ElasticsearchMeterRegistryElasticsearch5IntegrationTest
+        extends AbstractElasticsearchMeterRegistryIntegrationTest {
+
+    @Override
+    protected String getVersion() {
+        return "5.6.15";
```

Comment:
do we want to still support ES 5? even Elastic doesn't...

```suggestion
        return "5.6.16";
```

Reply:
@adriancole It sounds reasonable to drop the test as Elasticsearch 5.6.x has been EOLed at 2019-03-11, especially given that this is targeted for 1.4.0.

@shakuzen What do you think?

Reply:
For the record, I agree we shouldn't test this if we don't plan to fix a problem found only in this version. And I say we generally shouldn't fix things only affecting a version no longer maintained.

Reply:
@shakuzen Thanks! I dropped the test.